packages/calcite-components/src/components/checkbox/checkbox.scss

```diff
@@ -58,13 +58,23 @@
 }
 
 .toggle {
-  @apply focus-base;
+  @apply focus-base relative;
 
   &:active,
   &:focus,
   &:focus-visible {
     @apply focus-outset;
   }
+
+  &::after {
+    content: "";
+    inset-block-start: 50%;
+    inset-inline-start: 50%;
+    min-block-size: 24px;
```

Comment:
I think you can use `min-block-size: theme("width.6")` so that it uses tokens.

Reply:
I'm wondering since `width.6` evalutes to a `rem` value if the 24px AA minimum could go under if the page's default font-size is reduced far enough.  Maybe this is something we document as a best practice not to do?

Reply:
Yeah thats a good point. Ill defer to others judgement. We could maybe store a const somewhere else for 24px minimum. Ideally, this 24px is stored somewhere in either a global CSS custom prop or sass var.